app/src/main/java/org/literacyapp/authentication/helper/DetectionHelper.java

```diff
@@ -154,6 +153,20 @@ private static synchronized void setScreenBrightness(Context context, double cur
         }
     }
 
+    private static synchronized void setDisplayTemperatureNight(){
```

Comment:
Can we rename this method to clarify what it does? For example `setDisplayTemperatureNightBrighter`, or perhaps pass the Kelvin value (4500 or 2500) as a parameter to the method?: `setDisplayTemperatureNight(int kelvinValue);`